cradle-core/src/main/java/com/exactpro/cradle/testevents/TestEventToStore.java

```diff
@@ -1,17 +1,17 @@
 /*
- * Copyright 2020-2023 Exactpro (Exactpro Systems Limited)
+ *  Copyright 2023 Exactpro (Exactpro Systems Limited)
  *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
  *
- *     http://www.apache.org/licenses/LICENSE-2.0
+ *      http://www.apache.org/licenses/LICENSE-2.0
  *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
```

Comment:
Why do we have an extra space before each line?

Reply:
I've updated IDE to the `Build #IC-231.9011.34, built on May 16, 2023` version and the `Version Control > Commit > Commit Check > Update copyright` option was enabled